.github/workflows/python-package.yml

```diff
@@ -27,11 +27,10 @@ jobs:
     - name: Install dependencies
       working-directory: .
       run: |
+        chmod +x CI/setter.sh
         CI/setter.sh
-    - name: Check imports with isort
-      run: isort $(git ls-files '*.py')
     - name: Check with flake8
-      run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      run: flake8 driver/ --count --select=E9,F63,F7,F82,E302,W391,W292 --show-source --statistics
```

Comment:
All commands: installing dependencies, generating code and running tests should be in one `ci/test.sh` script.